lib/config/target.js

```diff
@@ -168,8 +169,8 @@ You can also more options via the 'target' option: 'browserslist' / 'browserslis
 	[
 		"[async-]node[X[.Y]]",
 		"Node.js in version X.Y. The 'async-' prefix will load chunks asynchronously via 'fs' and 'vm' instead of 'require()'. Examples: node14.5, async-node10.",
-		/^(async-)?node(\d+(?:\.(\d+))?)?$/,
-		(asyncFlag, major, minor) => {
+		/^(async-)?node((\d+)(?:\.(\d+))?)?$/,
+		(asyncFlag, _, major, minor) => {
```

Comment:
Bugfix, before when you have `14.18` - major was `14.18` and minor was `18`, so if you have `node14.8` in your target, you always have true, same here for other places

Reply:
how this work before 🙈

Reply:
Looks like nobody use it before :smile: